data/human/news.txt

```diff
@@ -1485,12 +1556,38 @@ news "syndicate pa system"
 			"Attention: baggage should not be left unattended. Any unattended baggage will be auctioned within days."
 			"Attention factory workers: please proceed to the fast-lane immigration desk. Time spent in regular immigration lanes will not be reimbursed."
 			"Dear spaceship captains: please be aware that our outfitters are open all hours of the day. There is never a reason to delay the purchase of your next piece of Syndicate equipment."
-			"Dear traveler: we recommend you to have dinner in our local restaurant 'The Wing'. You will not be disappointed."
+			"Dear traveler: we recommend you to have dinner in our local restaurant '${restaurant names}'. You will not be disappointed."
```

Comment:
Just nit picking here, but would a similar line to this one for a pub be suitable as well? Something like:

`"Dear traveller: we recommend you to have dinner in our local pub '${restaurant names}'. You will not be disappointed."`